src/components/Charts/ChartCaseDensity.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { FunctionComponent } from 'react';
+import { isDate } from 'lodash';
+import { min as d3min, max as d3max } from 'd3-array';
+import { curveNatural } from '@vx/curve';
+import { GridRows } from '@vx/grid';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { Area } from '@vx/shape';
+import { ParentSize } from '@vx/responsive';
+import { Column } from 'common/models/Projection';
+import { CASE_DENSITY_LEVEL_INFO_MAP } from 'common/metrics/case_density';
+import { LevelInfoMap } from 'common/level';
+import { formatUtcDate, formatDecimal } from 'common/utils';
+import { AxisBottom, AxisLeft } from './Axis';
+import ChartContainer from './ChartContainer';
+import RectClipGroup from './RectClipGroup';
+import ZoneAnnotation from './ZoneAnnotation';
+import Tooltip from './Tooltip';
+import LinePathRegion from './LinePathRegion';
+import * as TooltipStyle from './Tooltip.style';
+import {
+  computeTickPositions,
+  getChartRegions,
+  getZoneByValue,
+  last,
+  getAxisLimits,
+} from './utils';
+import * as Style from './Charts.style';
+
+type Point = {
+  x: number;
+  y: number;
+};
+
+const getDate = (d: Point) => new Date(d.x);
+const getY = (d: Point) => d?.y;
+
+/**
+ * Since the case density is already calculated with 1%, we need to scale
+ * accordingly to calculate the case density with a different mortality
+ * rate m%.
+ *
+ * case density (1%) = (7 day moving avg of deaths / 1%) / (population / 100k)
+ *
+ * case density (m%) = (7 day moving avg of deaths / m%) / (population / 100k)
+ *       = (m% / 1%) * (7 day moving avg of deaths / m%) / (population / 100k)
+ *       = (m% / 1%) * case density (1%)
```

Comment:
These last two lines don't seem right and don't match the code below (the code does `(1% / m%) * case density (1%)`, right?)

I guess it should maybe be:

> = (1% / m%) * (7 day moving avg of deaths / 1%)) / (population / 100k)
> = (1% / m%) * case density (1%)

But honestly I'm not sure if including the derivation here is all that helpful.  And it'd be better to just move this logic into Projection.ts if we can so we don't need to convert back/forth.

Reply:
Oops, right, the coding is what I meant — it makes sense to move it to Projections.ts 👍